### DISCUSSION TITLE
Setting the input 'max_distance' of 'geodesic_distance' function.

### DISCUSSION THREAD

**Discussion by GostInShell** [pyg-team/pytorch_geometric#2224] [Q&A]
I use the `geodesic_distance` function to compute geometrci distance between `source` and `dest` vertices pairs. It is said in the [documentation ](https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/utils/geodesic.html) that when `max_distance` is set, then only yields results for geodesic distances less than :obj:`max_distance`. However, when testing I find that settign `max_distance=0` and `max_distance=None` output exact the same results, while the former is much faster. Is this usual? Is the faster result less accurate?

**Comment by rusty1s** [comment]
Note that we are using `gdist` for geodesic distance computation, so it might be best to ask for implementation specific questions [here](https://github.com/the-virtual-brain/tvb-gdist). In case both functions yield the expected result, I would simply go with the faster one :)
